spec/suites/layer/GeoJSONSpec.js

```diff
@@ -516,3 +516,25 @@ describe("L.LayerGroup#toGeoJSON", function () {
 		});
 	});
 });
+describe("L.GeoJSON functions", function () {
+	it("L.GeoJSON.latLngToCoords", function () {
+		var arrLatLng = [-1.4837191022531273, 43.49222084042808];
+		var arrLngLat = [arrLatLng[1], arrLatLng[0]];
+		var latLng = L.latLng(arrLatLng);
+
+		expect(L.GeoJSON.latLngToCoords(latLng)).to.eql([43.49222084042808, -1.4837191022531273]);
+		expect(L.GeoJSON.latLngToCoords(latLng, 0)).to.eql([43, -1]);
+		expect(L.GeoJSON.latLngToCoords(latLng, null)).to.eql(arrLngLat);
+	});
+	
+	it("L.GeoJSON.coordsToLatLngs", function () {
+		var arrEN = [-1.4837191022531273, 43.49222084042808];
```

Comment:
Better keep variable names consistent between different tests.
Why rename to arrEN?

Reply:
Also, if you add more tests, it may make sense to share common data between them.
See for example how used `beforeEach` and `afterEach` in other test files.